arviz/__init__.py

```diff
@@ -4,13 +4,43 @@
 
 import logging
 import os
+import warnings
+import datetime
+from pathlib import Path
 
 from matplotlib.colors import LinearSegmentedColormap
 from matplotlib.pyplot import style
 import matplotlib as mpl
 from packaging import version
 
 
+def warn_once_per_day():
+    warning_dir = Path.home() / "arviz_data"
+    warning_dir.mkdir(exist_ok=True)
+
+    stamp_file = warning_dir / f"daily_warning"
```

Comment:
```suggestion
    stamp_file = warning_dir / "daily_warning"
```

code-wise this is the only comment, and I didn't actually see at first, it was pylint that flagged it